src/Components/CreateImageWizard/formComponents/Oscap.js

```diff
@@ -131,13 +131,13 @@ const AddPolicy = ({ input }) => {
   const [wantsPolicy, setWantsPolicy] = useState(oscapPolicy !== undefined);
   return (
     <>
-      <FormGroup label="Compliance policy">
+      <FormGroup label="Compliance profile">
```

Comment:
Wonder if we should name is 'oscap profile' just to be sure.

Reply:
yup!